util/tlgen/xbar.sim.core.tpl

```diff
@@ -0,0 +1,33 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# xbar_${xbar.name}_sim core file generated by `tlgen.py` tool
+name: "lowrisc:dv:xbar_${xbar.name}_sim:0.1"
+description: "XBAR DV sim target"
+filesets:
+  files_dv:
+    depend:
+      - lowrisc:ip:tlul:0.1
+      - lowrisc:ip:xbar_${xbar.name}
+      - lowrisc:dv:xbar_${xbar.name}_param_pkg
+      - lowrisc:dv:dv_utils
+      - lowrisc:dv:tl_agent
+      - lowrisc:dv:scoreboard
+      - lowrisc:dv:mem_model
+      - lowrisc:dv:xbar_env
+      - lowrisc:dv:xbar_test
+    files:
+      - ../../../../../dv/sv/xbar_dv/tb/xbar_macros.svh
+      - ../../../../../dv/sv/xbar_dv/tb/tb.sv
```

Comment:
can we make a core file to include these dependencies? afaik, we should try to eliminate relative paths that reference files outside of the directory where this core file is located in - otherwise this can create issues later on in the build flow.

Reply:
Make sense. Refactored the core file structure to remove the relative paths.